backend/routes/eventRoutes.js

```diff
@@ -9,16 +9,24 @@ router.get('/', async (req, res) => {
 })
 
 router.post('/', async (req, res) => {
-  const {  rating, location, name, duration, description, image, category  } = req.body
+  const {  rating, location, name, duration, description, image, category, userId, date, time ,lat, long } = req.body
+  const floatLat = parseFloat(lat);
+  const floatLong = parseFloat(long);
   
   const newEvent = await prisma.event.create({
     data: {
+      rating, 
```

Comment:
How is rating calculated at event creation time?

Reply:
At creation time the rating is 0.
**_To do:_** create rating system